examples/kv_store/src/util.rs

```diff
@@ -2,8 +2,11 @@ use std::time::Duration;
 
 pub const BUFFER_SIZE: usize = 10000;
 pub const ELECTION_TICK_TIMEOUT: u64 = 5;
-pub const TICK_PERIOD: Duration = Duration::from_millis(10);
+pub const OMNI_PAXOS_TICK_PERIOD: Duration = Duration::from_millis(10);
```

Comment:
```suggestion
pub const TICK_PERIOD: Duration = Duration::from_millis(10);
```

Reply:
could not make it under feature flg:
```rust
pub const UI_TICK_PERIOD: Duration = Duration::from_millis(200);
```
because the `tokio::select!` dose not supply feature inside, so the ui tick need to be here no matter ui started or not:
```rust
// examples/kv_store/src/server.rs
loop {
            tokio::select! {
                biased;
               // #[cfg(feature = "with_omnipaxos_ui")]    // this is not supplied
                _ = op_ui_tick_interval.tick() => {
                    #[cfg(feature = "with_omnipaxos_ui")]
                    self.omni_paxos_ui.tick(self.omni_paxos.lock().unwrap().get_ui_states());
                },
            }
        }
```